app/queries/coach_notes_resolver.rb

```diff
@@ -0,0 +1,19 @@
+class CoachNotesResolver < ApplicationQuery
+  property :student_id
+
+  def coach_notes
+    CoachNote.not_archived.where(student_id: student_id).includes(author: { avatar_attachment: :blob }).order('created_at DESC').limit(20)
```

Comment:
Maybe write this as `student.coach_notes.not_archived...`.

i.e., beginning scoping the relation from a known point?